github/resource_github_membership_test.go

```diff
@@ -29,6 +30,27 @@ func TestAccGithubMembership_basic(t *testing.T) {
 	})
 }
 
+func TestAccGithubMembership_caseInsensitive(t *testing.T) {
+	if len(testCollaborator) == 0 {
```

Comment:
Nit when checking for empty strings its best to explicitly check for an empty string so that future readers know the variable is a string and not some type that implements the length method. 

```suggestion
	if testCollaborator == "" {
```